### PR TITLE
Remove protoc regeneration check from Travis

### PR DESCRIPTION
**Pull request by codingllama**
Sometimes protoc will generate slightly different file descriptors for protos.
That's enough to make Travis fail. Since we don't have a better option (for
now), I've disabled diff checks for .pb.go files.

### CODE REVIEW

**Comment by codingllama** [comment]
FWIW, mockgen is not deterministic either. If there are conflicting imports the names of the aliases change between invocations. We've side stepped that bullet so far, so I've left it intact in Travis.

File: .travis.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ script:
   - ./scripts/presubmit.sh
   - |
       # Check re-generation didn't change anything
-      status=$(git status --porcelain) || :
+      status=$(git status --porcelain | grep -v .pb.go) || :
```
Comment (AlCutter) [inline .travis.yml]: Might be worth popping a comment in here as to why we're ignoring the PBs?

Comment (codingllama) [inline .travis.yml]: Good call, done.

File: .travis.yml [hunk 2]
```diff
@@ -43,7 +43,9 @@ script:
   - ./scripts/presubmit.sh
   - |
       # Check re-generation didn't change anything
-      status=$(git status --porcelain) || :
+      # Skip protoc-generated files (.pb.go) because protoc is not determinist
```
Comment (daviddrysdale) [inline .travis.yml]: nit for the next time you're nearby: "determinist" -> "deterministic"

Comment (codingllama) [inline .travis.yml]: Good call. #446.

**Pull request merged**